### PR TITLE
feat(tutorial): complete step 3

### PR DESCRIPTION
**Pull request by vladbalanescu**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -28,42 +31,118 @@ const headers = [
...
+  const [totalItems, setTotalItems] = useState(0);
+  const [firstRowIndex, setFirstRowIndex] = useState(0);
+  const [currentPageSize, setCurrentPageSize] = useState(10);
   return (
     <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
       <div className="bx--row repo-page__r1">
         <div className="bx--col-lg-16">
-          <RepoTable headers={headers} rows={rows} />
+          <Query query={REPO_QUERY}>
+            {({ loading, error, data: { organization } }) => {
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: Your repo page is not loading and throwing errors. 

Try changing this line to `{({ loading, error, data: { organization } }) => {`

File: src/content/RepoPage/RepoPage.js [hunk 2]
```diff
@@ -28,42 +31,118 @@ const headers = [
...
+                    columnCount={headers.length + 1}
+                    rowCount={10}
+                    headers={headers}
+                  />
+                );
+              // Something went wrong with the data fetching
+              if (error) return `Error! ${error.message}`;
+
+              // If we're here, we've got our data!
+              const { repositories } = organization;
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: Your repo page is not loading and throwing errors.

Try changing this to `data.organizations`

Comment (vladbalanescu) [reply src/content/RepoPage/RepoPage.js]: I am not sure what you mean by that. `data` is not defined in this context

**Comment by vladbalanescu** [comment]
@kellychurchill I see what's going on here: It is my access token which returns a 401 
image. Fixing this now

**Comment by vladbalanescu** [comment]
Now this is working for me locally and it should load fine

File: src/content/RepoPage/RepoPage.js (hunk 2 above)
Comment (kellychurchill) [reply src/content/RepoPage/RepoPage.js]: It is defined on line 102.

**Comment by kellychurchill** [comment]
Still seeing an error on your PR's Netlify preview: 
https://deploy-preview-2894--carbon-tutorial.netlify.com/#/repos

**Comment by vladbalanescu** [comment]
@kellychurchill That's because my `env.local` is not included in this one, that's what I get locally as well. Let me try adding this one in.

**Comment by vladbalanescu** [comment]
So basically my access token got revoked once pushed to github, so I need to generate a new one. that is getting undefined because there is no data, as my access token cannot be pushed to github basically, locally it works just fine @kellychurchill

**Comment by kellychurchill** [comment]
@vladbalanescu Everyone's tokens work the same way. 

Try making the changes I noted above.

**Comment by vladbalanescu** [comment]
@kellychurchill I understand that, but how are you expecting that data object to come as defined as long as it doesn't have an access token to access my github repo? So that works fine locally for me, but once I push that `.env.local` file is gone so the netfly robot is not able to access that page, hence that object gets `undefined`.

**Comment by vladbalanescu** [comment]
@kellychurchill Ok, I added the required change in, I am not sure what difference that's going to make, it is working fine locally, just checked it now, waiting for the bot to finish the build and see how that's performing on here

**Pull request closed without merging**